src/test/java/de/tum/in/www1/artemis/course/CourseTestService.java

```diff
@@ -548,11 +548,6 @@ public void testCreateCourseWithDefaultChannels() throws Exception {
             var channels = channelRepository.findChannelsByCourseId(course2.getId());
             assertThat(channels).hasSize(DefaultChannelType.values().length);
             channels.forEach(channel -> assertThat(Arrays.stream(DefaultChannelType.values()).map(DefaultChannelType::getName)).contains(channel.getName()));
-            // Check if newly added instructor and student was added to default channels
-            channels.forEach(channel -> {
-                var participants = conversationParticipantRepository.findConversationParticipantByConversationId(channel.getId());
-                assertThat(participants).hasSize(2); // 1 instructor and 1 student
```

Comment:
But tutors and instructors are still registered?

https://github.com/ls1intum/Artemis/blob/cbf79af2126fa0e0dd8f313d538e6982933177a9/src/main/java/de/tum/in/www1/artemis/service/metis/conversation/ChannelService.java#L183

Can we assert `hasSize(1)`?

Reply:
No, this is only used for exam channels and not default course channels, so no participants are added in this case here